composites/Plugin/Shared/components/KeywordInput.js

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import styled from "styled-components";
+import colors from "../../../../style-guide/colors.json";
+import PropTypes from "prop-types";
+
+let KeywordField = styled.input`
+	margin-right: 0.5em;
+	border-color: ${ props => props.borderColor };
+`;
+
+const ErrorText = styled.div`
+	font-size: 1em;
+	color: ${ colors.$color_red};
+	margin: 1em 0;
+	min-height: 1.8em;
+`;
+
+class KeywordInput extends React.Component {
+	/**
+	 * Constructs a KeywordInput component
+	 *
+	 * @param {Object} props The props for this input field component.
```

Comment:
I would love some extra documentation here on the props themselves (id, onChange, label and keyword).